src/lib/y2partitioner/widgets/blk_table.rb

```diff
@@ -0,0 +1,91 @@
+require "yast"
+
+require "cwm/table"
+
+require "y2partitioner/icons"
+
+module Y2Partitioner
+  module Widgets
+    # Table widget to represent given list of Block Devices.
+    class BlkTable
```

Comment:
well, I try to be consisten with yast2-storage-ng. Which also use BlkDevice. Maybe we can reference it and have `BlkDevicesTable`?